infra-as-code/bicep/orchestration/mgDiagSettingsAll/mgDiagSettingsAll.bicep

```diff
@@ -84,5 +84,6 @@ module modMgChildrenDiagSet '../../modules/mgDiagSettings/mgDiagSettings.bicep'
 module modCustomerUsageAttribution '../../CRML/customerUsageAttribution/cuaIdTenant.bicep' = if (!parTelemetryOptOut) {
   #disable-next-line no-loc-expr-outside-params //Only to ensure telemetry data is stored in same location as deployment. See https://github.com/Azure/ALZ-Bicep/wiki/FAQ#why-are-some-linter-rules-disabled-via-the-disable-next-line-bicep-function for more information //Only to ensure telemetry data is stored in same location as deployment. See https://github.com/Azure/ALZ-Bicep/wiki/FAQ#why-are-some-linter-rules-disabled-via-the-disable-next-line-bicep-function for more information
   name: 'pid-${varCuaid}-${uniqueString(deployment().location)}'
+  scope: tenant()
   params: {}
 }
```

Comment:
Change this module out to use the management group scoped level version here https://github.com/Azure/ALZ-Bicep/blob/main/infra-as-code/bicep/CRML/customerUsageAttribution/cuaIdManagementGroup.bicep